ops/testnet/prod/backend/config.tf

```diff
@@ -40,7 +40,8 @@ locals {
       }
     }
     environment = var.stage
-    cache = {
+    network     = var.environment
```

Comment:
kinda weird that `stage == environment` and `environment == network`

doesn't really seem consistent with the other env var usage?